plugin/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go

```diff
@@ -359,6 +359,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 			informerFactory.Apps().V1beta1().StatefulSets(),
 			informerFactory.Core().V1().Services(),
 			v1.DefaultHardPodAffinitySymmetricWeight,
+			true,
```

Comment:
Add a comment what this sets (I know I was pretty bad about this, but we probably should start annotating literals such as this).